lib/opencensus/trace/integrations/rails.rb

```diff
@@ -91,24 +73,7 @@ module Integrations
       #     config.opencensus.trace.middleware_placement = ::Rails::Rack::Logger
       #
       class Rails < ::Rails::Railtie
-        ##
-        # The ActiveSupport notifications that will be reported as spans by
-        # default. To change this list, update the value of the
-        # `trace.notifications.events` configuration.
-        #
-        DEFAULT_NOTIFICATION_EVENTS = [
-          "sql.active_record",
-          "render_template.action_view",
-          "send_file.action_controller",
-          "send_data.action_controller",
-          "deliver.action_mailer"
-        ].freeze
-
         OpenCensus::Trace.configure do |c|
-          c.add_config! :notifications do |rc|
-            rc.add_option! :events, DEFAULT_NOTIFICATION_EVENTS.dup
-            rc.add_option! :attribute_namespace, "rails/"
-          end
```

Comment:
I'm suggesting that we move this to it's own Railtie.

I found that it was necessary if you trace both Sidekiq and Rack because if you register with `ActiveSupport::Notifications.subscribe` in both the Sidekiq and Rack code you get duplicate ActiveSupport traces